lib/widget/video.dart

```diff
@@ -77,41 +141,104 @@ class MyController extends WidgetController {
             _playerController!.value.duration) {
           notifyListeners();
         }
+        playerCapabilities?.videoPlayerStateChange();
       });
     }
   }
 }
 
-class VideoState extends WidgetState<Video> {
+mixin PlayerCapabilities on WidgetState<Video> {
+  void stop();
+  void pause();
+  void play();
+  void mute();
+  void unmute();
+  void seekTo(dynamic value);
+  void setPlaybackRate(double speed);
+  void setVolume(double volume);
+  void videoPlayerStateChange();
+}
+
+class VideoState extends WidgetState<Video> with PlayerCapabilities {
+  @override
+  void didChangeDependencies() {
+    widget._controller.playerCapabilities = this;
```

Comment:
this can cause memory leak as you are not setting to null in dispose of the state as a result the last state can hang around in memory. 

widget._controller.playerCapabilities = null; inside dispose() should fix it

Reply:
Done